assessments/projects/spiffe-spire/outline.md

```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
 
 ## Background
 
-SPIFFE and SPIRE are a set of APIs and associated tooling that provides a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
+Secure Protection Identity Framework for Everyone (SPIFFE) are a set of APIs with SPIRe as the production grade implemtnation and associated tooling that provide a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
 
 _Maturity_
 
 (Examples)
-- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exclusive list and it’s likely that due to the secrecy of a security framework other organizations are also SPIFFE and SPIRE users privately.
+- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exhaustive list, due to the sensitivity surrounding exposure of security frameworks in use it is likely other organizations are also SPIFFE and SPIRE users privately.
```

Comment:
hmm maybe rephrase this to

"This is not an exhaustive list as other organizations may use SPIFFE and SPIRE privately (due to the sensitivity surrounding exposure of security frameworks being used)"

Reply:
love that wording better!